tasks/broad/ImputationTasks.wdl

```diff
@@ -5,17 +5,18 @@ task CalculateChromosomeLength {
     File ref_dict
     Int chrom
     String ubuntu_docker
```

Comment:
I believe we discussed defining the default docker definition here and removing the block in imputation.wdl
```suggestion
    String ubuntu_docker = "ubuntu:20.04"
```